lib/ddtrace/transport.rb

```diff
@@ -20,6 +23,17 @@ class HTTPTransport
     TRACE_COUNT_HEADER = 'X-Datadog-Trace-Count'.freeze
     RUBY_INTERPRETER = RUBY_VERSION > '1.9' ? RUBY_ENGINE + '-' + RUBY_PLATFORM : 'ruby-' + RUBY_PLATFORM
 
+    HEADER_META_LANG = 'Datadog-Meta-Lang'.freeze
+    HEADER_META_LANG_VERSION = 'Datadog-Meta-Lang-Version'.freeze
+    HEADER_META_LANG_INTERPRETER = 'Datadog-Meta-Lang-Interpreter'.freeze
+    HEADER_META_TRACER_VERSION = 'Datadog-Meta-Tracer-Version'.freeze
```

Comment:
We may leverage metrics tags to aggregate the value by all these `HEADER_META_*` keys. We can start with that for not and expand later.

Reply:
As @pawelchcki suggested, we can use also the content-type as a tag (json / msgpack)